### PR TITLE
Basic Storage role

### PR DESCRIPTION
**Pull request by etsauer**
#### What does this PR do?

Role to set up storage
#### How should this be manually tested?

Follow README included in role
#### Is there a relevant Issue open for this?

n/a
#### Who would you like to review this?

/cc @oybed  @vvaldez

### CODE REVIEW

File: rhc-ose-ansible/roles/storage/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# storage Ansible Role
+
+Configures Disk partitions, VGs, LVs, filesystem, and mount points
```
Comment (oybed) [inline rhc-ose-ansible/roles/storage/README.md:3]: The current implementation doesn't seem to support more than just one partition, which may be fine, but personally I'd like to see support for handling more partitions (both primary and extended). However, if you choose to keep it to just one partition for now, it should be noted in the README as a highlighted "note:" line as the line above says "partitions" and hence may not be quite accurate.

File: rhc-ose-ansible/roles/storage/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+```
+
+Create a host groups var at {inventory_dir}/group_vars/storage_hosts like:
+
+[source,yaml]
+```
+volume_groups:
+- disk: /dev/vdb
+  vg: vg_sat6
+  fstype: xfs
```
Comment (oybed) [inline rhc-ose-ansible/roles/storage/README.md:32]: Probably should remove the "fstype" here as it's not applied to a VG

**Comment by oybed** [comment]
Looks good for the most part. A few comments in the code. In addition, I'm wondering if we couldn't just make some steps optional so they are not all required. For example, this could be useful for those cases where you may want to create a disk partition, and maybe a PV and a VG, but not a LV or filesystem. In that case, you'd only specify the volume group and physical disk, but no logical volumes. Just a though to make it a bit more flexible and hence usable for other scenarios.

**Comment by etsauer** [comment]
@oybed would you mind proposing what you would like to see supported via a change to the group_vars example, and I will make it work? for example, do you want sections for each item? partitions, filesystems, etc? what parameters would you want in each?

As far as making each step "optional" i will go ahead and do that, however the complication will come in when, say we want to only create a partition, skip the vg/lv, but still create a mount point, or some other combination. What should we support in that respect?

**Pull request closed without merging**